src/beam.ts

```diff
@@ -68,26 +68,27 @@ export class Beam extends Element {
   postFormatted: boolean;
   slope: number = 0;
 
-  readonly #stemDirection: number;
-  readonly #ticks: number;
+  private readonly _stemDirection: number;
+  private readonly _ticks: number;
 
-  #yShift: number = 0;
-  #breakOnIndexes: number[];
-  #beamCount: number;
-  #unbeamable?: boolean;
+  protected yShift: number = 0;
+  private breakOnIndexes: number[];
+  private _beamCount: number;
```

Comment:
what is the meaning of the underscore?

Reply:
If there was an underscore before I restored it. Mostly when there was a getter or setter for the property that conflicted (like text and width). Since the PR that put these in wasn't cleanly resolved it was hard to tell what the case was. This looked like a prime case where the attribute is likely to be replaced by a property that computes beam counts on the fly. But I'll happily replace it back if it seems confusing. But isn't beamCount also a method?